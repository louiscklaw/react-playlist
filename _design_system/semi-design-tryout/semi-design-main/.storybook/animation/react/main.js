const config = require('../../base/base');

module.exports = {
  ...config,
  "stories": [
    '../../../packages/(semi-animation-react|semi-animation-styled)/_story/*.react.stories.(js|jsx)',
  ],
  typescript: {
    check: false,
    checkOptions: {}
  },
};
