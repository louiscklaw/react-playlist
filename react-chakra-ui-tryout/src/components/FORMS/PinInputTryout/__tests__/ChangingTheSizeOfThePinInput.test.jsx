import React from 'react';
import { render } from '@testing-library/react';
import renderer from 'react-test-renderer';
import { ChakraProvider } from '@chakra-ui/react';

import { ChangingTheSizeOfThePinInput } from '../';

describe('test ChangingTheSizeOfThePinInput tryout', () => {
  test('helloworld', () => {
    const tree = renderer.create(
      <ChakraProvider>
      <ChangingTheSizeOfThePinInput />
      </ChakraProvider>
      ).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
