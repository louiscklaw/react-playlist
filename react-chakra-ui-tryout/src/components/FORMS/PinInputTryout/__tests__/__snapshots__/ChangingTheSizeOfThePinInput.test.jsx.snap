// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test ChangingTheSizeOfThePinInput tryout helloworld 1`] = `
Array [
  <div
    className="chakra-stack css-n21gh5"
  >
    <div
      className="chakra-stack css-84zodg"
    >
      <input
        aria-label="Please enter your pin code"
        autoComplete="off"
        className="chakra-pin-input css-kvmxzd"
        id="pin-input-1--1"
        inputMode="numeric"
        onBlur={[Function]}
        onChange={[Function]}
        onFocus={[Function]}
        onKeyDown={[Function]}
        placeholder=""
        type="tel"
        value=""
      />
      <input
        aria-label="Please enter your pin code"
        autoComplete="off"
        className="chakra-pin-input css-kvmxzd"
        id="pin-input-1--1"
        inputMode="numeric"
        onBlur={[Function]}
        onChange={[Function]}
        onFocus={[Function]}
        onKeyDown={[Function]}
        placeholder=""
        type="tel"
        value=""
      />
      <input
        aria-label="Please enter your pin code"
        autoComplete="off"
        className="chakra-pin-input css-kvmxzd"
        id="pin-input-1--1"
        inputMode="numeric"
        onBlur={[Function]}
        onChange={[Function]}
        onFocus={[Function]}
        onKeyDown={[Function]}
        placeholder=""
        type="tel"
        value=""
      />
    </div>
    <div
      className="chakra-stack css-84zodg"
    >
      <input
        aria-label="Please enter your pin code"
        autoComplete="off"
        className="chakra-pin-input css-1pqxzd4"
        id="pin-input-2--1"
        inputMode="numeric"
        onBlur={[Function]}
        onChange={[Function]}
        onFocus={[Function]}
        onKeyDown={[Function]}
        placeholder=""
        type="tel"
        value=""
      />
      <input
        aria-label="Please enter your pin code"
        autoComplete="off"
        className="chakra-pin-input css-1pqxzd4"
        id="pin-input-2--1"
        inputMode="numeric"
        onBlur={[Function]}
        onChange={[Function]}
        onFocus={[Function]}
        onKeyDown={[Function]}
        placeholder=""
        type="tel"
        value=""
      />
      <input
        aria-label="Please enter your pin code"
        autoComplete="off"
        className="chakra-pin-input css-1pqxzd4"
        id="pin-input-2--1"
        inputMode="numeric"
        onBlur={[Function]}
        onChange={[Function]}
        onFocus={[Function]}
        onKeyDown={[Function]}
        placeholder=""
        type="tel"
        value=""
      />
    </div>
    <div
      className="chakra-stack css-84zodg"
    >
      <input
        aria-label="Please enter your pin code"
        autoComplete="off"
        className="chakra-pin-input css-17u1q9k"
        id="pin-input-3--1"
        inputMode="numeric"
        onBlur={[Function]}
        onChange={[Function]}
        onFocus={[Function]}
        onKeyDown={[Function]}
        placeholder=""
        type="tel"
        value=""
      />
      <input
        aria-label="Please enter your pin code"
        autoComplete="off"
        className="chakra-pin-input css-17u1q9k"
        id="pin-input-3--1"
        inputMode="numeric"
        onBlur={[Function]}
        onChange={[Function]}
        onFocus={[Function]}
        onKeyDown={[Function]}
        placeholder=""
        type="tel"
        value=""
      />
      <input
        aria-label="Please enter your pin code"
        autoComplete="off"
        className="chakra-pin-input css-17u1q9k"
        id="pin-input-3--1"
        inputMode="numeric"
        onBlur={[Function]}
        onChange={[Function]}
        onFocus={[Function]}
        onKeyDown={[Function]}
        placeholder=""
        type="tel"
        value=""
      />
    </div>
    <div
      className="chakra-stack css-84zodg"
    >
      <input
        aria-label="Please enter your pin code"
        autoComplete="off"
        className="chakra-pin-input css-17dn0h8"
        id="pin-input-4--1"
        inputMode="numeric"
        onBlur={[Function]}
        onChange={[Function]}
        onFocus={[Function]}
        onKeyDown={[Function]}
        placeholder=""
        type="tel"
        value=""
      />
      <input
        aria-label="Please enter your pin code"
        autoComplete="off"
        className="chakra-pin-input css-17dn0h8"
        id="pin-input-4--1"
        inputMode="numeric"
        onBlur={[Function]}
        onChange={[Function]}
        onFocus={[Function]}
        onKeyDown={[Function]}
        placeholder=""
        type="tel"
        value=""
      />
      <input
        aria-label="Please enter your pin code"
        autoComplete="off"
        className="chakra-pin-input css-17dn0h8"
        id="pin-input-4--1"
        inputMode="numeric"
        onBlur={[Function]}
        onChange={[Function]}
        onFocus={[Function]}
        onKeyDown={[Function]}
        placeholder=""
        type="tel"
        value=""
      />
    </div>
  </div>,
  <span />,
]
`;
